### ISSUE TITLE
Pool details - MVP v1

### ISSUE THREAD

**Issue by EstherCentrifuge** [centrifuge/apps#642]
[Specis in HackMD](https://centrifuge.hackmd.io/1OiMlCCWTL2YwUN8c_sFnA?both#Pool-details)
[Figma node](https://www.figma.com/file/d1bEBkwllUQe27nQM6VlJu/Centrifuge-app?node-id=1971%3A54008) 

**Open points:**
- Visuals on Figma might get updated within the first 2 das of ticket creation

Summary 
Pool details section with the create pool flow to fill in/ upload / select the following details:
- Pool name
- Pool icon
- Asset class
- Currency
- Initial Maximum reserve
- Minimum epoch duration
- Challenge time

**Comment by EstherCentrifuge** [comment]
Needs to be added to Create & manage pool(s) - MVP v1 Epic @denniswell

**Comment by EstherCentrifuge** [comment]
thanks!

**Comment by onnovisser** [comment]
@EstherCentrifuge  Couple questions re the spec: 
- Is there a maximum "Minimum epoch duration" we want to allow? 
- Same as above for "Challenge time"
- Pool icon mentions a size of 40x40px, but SVGs are infinitely scalable, so mentioning a size doesn't really make sense to me
- Is there a design for how the tooltips should look, and is there a spec as to the text they should contain?

**Comment by EstherCentrifuge** [comment]
@onnovisser here is the tooltip design:
https://www.figma.com/file/TPFzwr1VVtg7MZlRVwMKvS/Fabric-design-system?node-id=964%3A2718

**Comment by armandomistral** [comment]
@onnovisser, you are right size about the size. Suggesting to use the label `Pool icon (SVG, square format)`. (cc. @EstherCentrifuge )

**Comment by EstherCentrifuge** [comment]
@onnovisser  I didn't state this before but now it's "important" can we ensure the pool image upload is mandatory please :)

**Comment by onnovisser** [comment]
sure thing

**Comment by armandomistral** [comment]
@onnovisser & @EstherCentrifuge, added the "required" flag to the label on mock https://www.figma.com/file/d1bEBkwllUQe27nQM6VlJu/Centrifuge-app?node-id=2825%3A62530.
Proposing wording 'Pool icon: SVG in square size (required)'

**Comment by EstherCentrifuge** [comment]
We removed the Challange time, and added minutes to the Epoch time @onnovisser 
Dono if that was clear.

**Comment by onnovisser** [comment]
I see it now. As users now won't set the challenge time, what should the default challenge time be when creating the pool? Also, same question for NAV age.
